web-storybook/.storybook/main.js

```diff
@@ -72,13 +72,6 @@ module.exports = {
             '../../web-marketplace/src/**/*.svg',
           ),
         }),
-        tsconfigPaths({
-          root: path.resolve(__dirname, '.'),
-          projects: [
-            path.resolve(__dirname, '../../web-components/tsconfig.json'),
-            path.resolve(__dirname, '../../web-marketplace/tsconfig.json'),
-          ],
-        }),
```

Comment:
Why don't we need this?

Reply:
I could not get `tsconfigPaths` to work with the `bun` migration (it's needed for absolute path to work in storybook). I think it's more related to our configuration rather than a bun-specific issue but as I was saying [here](https://github.com/regen-network/regen-web/pull/2139#issuecomment-1798621727) this is not blocking and we can investigate it further in a follow-up issue.